### DISCUSSION TITLE
How did you create this?

### DISCUSSION THREAD

**Discussion by dorianim** [af/hyperpaper-planner#4] [Q&A] [answered]
Hi,

I love this pdf, it's a great idea!
Just one question: How did you create it?

Regards
Dorian

**Comment by af** [comment, marked as answer]
Just responded to this on Reddit, copying my answer:

> Love and duct tape 😂 Seriously though, I looked at some desktop publishing software initially but as a software engineer I found it easier to make it programmatically (in typescript)
